.nf-core.yml

```diff
@@ -1,4 +1,5 @@
 repository_type: pipeline
+nf_core_version: "2.14.0"
 lint:
   files_unchanged:
-    - .github/CONTRIBUTING.md
+    - .github/CONTRIBUTING.md
```

Comment:
why this file?

Reply:
Don't know. I think it was just part of the template-update.

Reply:
The template update modified only the new line. Adding this file will skip the linting test. You should do this only if you edited `CONTRIBUTING.md` and don't want to lint it.

Reply:
I guess somebody put `.github/CONTRIBUTING.md` in `.nf-core.yml` because it was failing the linting test.

Reply:
ok let's deal with that in the other PR then